protocol/src/lib.rs

```diff
@@ -450,11 +450,11 @@ impl PacketWriter {
 #[derive(Clone)]
 pub struct PacketHandler {
     /// A unique identifier for the communication session.
-    pub session_id: [u8; 32],
```

Comment:
I was thinking it might be easier to have the field just be public, but I don't mind this approach either